backend/src/services/master/master.entity.ts

```diff
@@ -4,55 +4,65 @@ class Master {
   public id: string;
   public name: string;
   public email: string;
+  public password: string;
```

Comment:
Master entity cannot contain password directly. It should contain passwordHash, passwordSalt. Password hash cannot be decrypted